app/src/main/java/com/hedvig/app/feature/home/service/HomeTracker.kt

```diff
@@ -0,0 +1,13 @@
+package com.hedvig.app.feature.home.service
+
+import com.mixpanel.android.mpmetrics.MixpanelAPI
+
+class HomeTracker(val mixpanel: MixpanelAPI) {
+    fun startClaimOutlined() = mixpanel.track("START_CLAIM_OUTLINED")
+    fun startClaimContained() = mixpanel.track("START_CLAIM_CONTAINED")
+    fun connectPayin() = mixpanel.track("CONNECT_PAYIN")
+    fun psaLink() = mixpanel.track("PSA_LINK")
+    fun emergency() = mixpanel.track("EMERGENCY")
+    fun openCommonClaimActivity() = mixpanel.track("OPEN_COMMON_CLAIM_ACTIVITY")
+    fun retry() = mixpanel.track("RETRY")
```

Comment:
Is this really the text key for this button? 🤔

Reply:
If you mean `openCommonClaimActivity()` then is because the the text key depends on what typ of common claim it is

Reply:
Our current strategy is to only track things by their text key. Things that do not have a determinable text key will not be tracked.

Reply:
Okey they are now removed. But what is the reasoning behind that?

Reply:
Consistency between platforms, more or less. We're trying to develop a better tracking plan, but for now this is what we're doing